leaf-ui/package-lock.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
What is the file for and do you still need "zebra_dialog" portion of it. It looks like you removed the rest of the zebra dialog files?

Reply:
Please comment the file.

Reply:
Also, should this file live in the plugins folder?

Reply:
Sorry, I removed the zebra_dialog. That was another library for customizing dialog, but I chose to use sweetalert instead. And package.json is not needed, so I will remove them and push it again.